src/content/docs/kubernetes-pixie/k8s-otel/install.mdx

```diff
@@ -0,0 +1,612 @@
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.hostmetrics.scrapeInterval`</td>
+            <td>String</td>
+            <td>Sets the scrape interval for the `hostmetrics` receiver.</td>
+            <td>`1m`</td>
+        </tr>
+        <tr>
+            <td>`receivers.kubeletstats.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `kubeletstats` receiver is enabled for collecting metrics from the Kubelet.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`receivers.kubeletstats.scrapeInterval`</td>
+            <td>String</td>
+            <td>Sets the scrape interval for the `kubeletstats` receiver.</td>
+            <td>`1m`</td>
+        </tr>
+        <tr>
+            <td>`receivers.filelog.enabled`</td>
+            <td>Boolean</td>
+            <td>Specifies whether the `filelog` receiver is enabled for collecting logs from files, typically container logs.</td>
+            <td>`true` or `false`</td>
+        </tr>
+        <tr>
+            <td>`lowDataMode`</td>
+            <td>Boolean</td>
+            <td>Enables a mode that sends only the essential metrics required to illuminate the New Relic Kubernetes user interface. This helps reduce data ingestion volume.</td>
+            <td>`true` or `false`</td>
+        </tr>
+    </tbody>
+</table>
+
+  </Collapser>
+</CollapserGroup>
+
+
+## Alerts [#alerts]
+
+You can install essential alert policies by going through the guided installation flow in Integrations & Agents. This automatically sets up alert policy named **Kubernetes (OpenTelemetry) alert policy** in your New Relic account with multiple alert conditions designed for Kubernetes observability. Customize these conditions to suit your specific monitoring needs.
+
+<img
+    title="Kubernetes OpenTelemetry alert policy"
+    alt="Kubernetes OpenTelemetry alert policy"
+    src="/images/alert-k8s-policy.webp"
+/>
+
+
+## Find and use data [#find]
+
+Check out these documents to learn more on how to find data:
+
+* [Explore your Kubernetes cluster](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/) to know the status of your cluster, from the control plane to nodes and pods.
+
+* [Kubernetes APM summary page](/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page/) which offers insights into your Kubernetes integration alongside your monitored applications.
+
+
+
+
+## Metrics [#metrics]
+
+* The `LowDataMode` is the default mode. To view the full list of metrics refer to [`LowDataMode` list](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/docs/metrics-lowDataMode.md).
+* [Metrics - Full list](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/docs/metrics-full.md)
+
+
+
+## Uninstall [#uninstall]
```

Comment:
Is it clear enough?
It looks like we need to execute 2 steps.

Should we change the titles to match what we have in the Install section?

Uninstall Helm chart ---> Helm Install
Uninstall manifests ---> Manifests install